src/osgPlugins/fbx/WriterCompareTriangle.cpp

```diff
@@ -1,9 +1,10 @@
 #include "WriterCompareTriangle.h"
 
-WriterCompareTriangle::WriterCompareTriangle(const osg::Geode& in_geode, unsigned int in_nbVertices):
-        geode(geode)
+WriterCompareTriangle::WriterCompareTriangle(const osg::Geode& geode,
+                                             unsigned int      nbVertices)
+                                             :    geode(geode)
 {
-    cutscene(in_nbVertices, geode.getDrawable(0)->asGeometry()->getBoundingBox());
+    cutscene(nbVertices, geode.getDrawable(0)->asGeometry()->getBoundingBox());
```

Comment:
Looks like we should use the 3.6 version of this file.

Reply:
Reverted changes to this file (pending)